nistats/second_level_model.py

```diff
@@ -0,0 +1,451 @@
+from warnings import warn
+import sys
+import time
+import pandas as pd
+import numpy as np
+from nibabel import Nifti1Image
+
+from sklearn.base import BaseEstimator, TransformerMixin, clone
+from nilearn._utils.niimg_conversions import check_niimg
+from nilearn._utils import CacheMixin
+from nilearn.input_data import NiftiMasker
+from patsy import DesignInfo
+
+from .first_level_model import FirstLevelModel
+from .regression import OLSModel, SimpleRegressionResults
+from .contrasts import compute_contrast
+from .utils import _basestring
+from .design_matrix import _create_second_level_design
+
+
+def run_glm(Y, X, n_jobs=1, verbose=0):
```

Comment:
We already have a run_glm in first-level_glm. As these two guys are doing the same thing, this one should be replaced by a call to the other one.